### PR TITLE
Very minor changes of some JS files

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/assets/javascripts/app/collections/posts.js [hunk 1]
```diff
@@ -5,4 +5,3 @@ app.collections.Posts = Backbone.Collection.extend({
   url : "/posts"
 });
 // @license-end
-
```
Comment (jhass) [inline app/assets/javascripts/app/collections/posts.js]: https://github.com/diaspora/diaspora/blob/develop/.editorconfig#L5

File: app/assets/javascripts/app/collections/comments.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ app.collections.Comments = Backbone.Collection.extend({
   make : function(text){
     var self = this;
 
-    var comment = new app.models.Comment({text: text });
+    var comment = new app.models.Comment({ 'text': text });
```
Comment (RyDroid) [inline app/assets/javascripts/app/collections/comments.js]: The other strings of the file use simple-quote. Do you have to change all the strings of this file?

File: app/assets/javascripts/app/collections/posts.js (hunk 1 above)
Comment (RyDroid) [inline app/assets/javascripts/app/collections/posts.js]: There is one...

File: app/assets/javascripts/app/collections/comments.js (hunk 2 above)
Comment (jhass) [inline app/assets/javascripts/app/collections/comments.js]: No, see the last FAQ entry: https://houndci.com/faq

Comment (jhass) [inline app/assets/javascripts/app/collections/comments.js]: No, see the last FAQ entry: https://houndci.com/faq

File: app/assets/javascripts/app/collections/posts.js (hunk 1 above)
Comment (jhass) [inline app/assets/javascripts/app/collections/posts.js]: Right, I guess the diff got me confused, sorry.

**Comment by denschub** [comment]
Yay, thank you!

(no backport - there are conflicts and there are no bugs fixed)

**Pull request merged**